components/devRelEvents/FeaturedCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { Flex, Icon, Image } from "@chakra-ui/react";
+import { useTrack } from "hooks/analytics/useTrack";
+import { FiCalendar } from "react-icons/fi";
+import { Badge, Heading, LinkButton, Text } from "tw-components";
+
+interface FeaturedCardProps {
+  type: string;
+  title: string;
+  date: string;
+  banner: string;
+  link: string;
+}
+
+export const FeaturedCard: React.FC<FeaturedCardProps> = ({
+  type,
+  title,
+  date,
+  banner,
+  link,
+}) => {
+  const trackEvent = useTrack();
+
+  return (
+    <Flex
+      pos="relative"
+      flexDir="column"
+      bg="whiteAlpha.100"
+      rounded="lg"
+      maxW="500px"
+    >
+      <Badge
+        pos="absolute"
+        top={4}
+        left={4}
+        bg="white"
+        color="black"
+        p={2}
+        rounded="md"
+      >
+        {type}
+      </Badge>
+      <Image src={banner} alt={title} roundedTop="lg" />
+      <Flex flexDir="column" p={4} align="center">
+        <Heading>{title}</Heading>
+        <Flex align="center" my={2}>
+          <Icon as={FiCalendar} color="gray.300" />
+          <Text ml={2} color="gray.300">
+            {date}
+          </Text>
+        </Flex>
+
+        <LinkButton
```

Comment:
Give this button a little more margin with the bottom of the card, it looks a little dense